.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node
+dist: trusty
+language: node_js
+node_js:
+  - "6.9.5"
+before_script:
+  - npm install
```

Comment:
Are you sure you need the `npm install` ? I believe you should not, please double check